tests/test_packageAlerts.py

```diff
@@ -153,6 +204,49 @@ def setUp(self):
         self.cutoutWcs.wcs.cd = self.exposure.getWcs().getCdMatrix()
         self.cutoutWcs.wcs.ctype = ["RA---TAN", "DEC--TAN"]
 
+    def _deserialize_alert(self, alert_bytes):
+        """Deserialize an alert message from Kafka.
+
+        Parameters
+        ----------
+        alert_bytes : `bytes`
+            Binary-encoding serialized Avro alert, including Confluent Wire
+            Format prefix.
+
+        Returns
+        -------
+        alert : `dict`
+            An alert payload.
+        """
+        schema = alertPack.Schema.from_uri(
+            str(alertPack.get_uri_to_latest_schema()))
+
+        header_bytes = alert_bytes[:5]
+        version = self._deserialize_confluent_wire_header(header_bytes)
+        self.assertEqual(version, 0)
```

Comment:
What is the purpose of this assert?

Reply:
Its a sanity check leftover to make sure the version was correct. I can remove it since it isn't necessary for the unit checks.